README.md

```diff
@@ -128,3 +128,40 @@ In addition, targets can compose through nesting:
 ```
 
 allowing splitting of `pkg/features/x.js` resolution between browser production and browser development environments.
+
+#### Not Found Target
+
+In addition to string targets it is also possible to define a not found target which indicates that there should be no mapping for the given module:
+
+```js
+{
+  "browser": false,
+  "exports": {
+    "./": { browser: false }
```

Comment:
IIRC the original/current import map syntax for this was `[]` or `null`. Any specific reason to go with `false` over either of those?

Reply:
Sure, I've updated this to use `null`. That may also be more compatible with existing use of `browser: false` actually.